boards/px4/fmu-v4/init/rc.board_defaults

```diff
@@ -0,0 +1,27 @@
+#!nsh
+#
+# PX4 FMUv4 specific board init
+#
+#------------------------------------------------------------------------------
+#
+# UART mapping on FMUv5:
+#
+# UART1         /dev/ttyS0      wifi
+# USART2        /dev/ttyS1      TELEM1 (flow control)
+# USART3        /dev/ttyS2      TELEM2 (flow control)
+# UART4
+# UART7                         CONSOLE
+# UART8         /dev/ttyS6      SERIAL4/TELEM4
```

Comment:
This is PX4IO on V5 full and a no connect on Mini

Reply:
Can you help do a pass to fix these? I'm not sure how many were correct to begin with.

Reply:
@dagar OK will do

Reply:
Actually, I'm not sure we should have these to begin with. I'd rather coordinate with the proper documentation and have a single source. https://docs.px4.io/en/flight_controller/pixracer.html

Reply:
@dagar - on second thought: I  recall having links to doc pages that disappear and and have loss of information. That is not sustainable and it is a contradiction to source code driven documentation. How about we extract this data to the doc pages? Then the information is were it is needed and the documentation is correct?

Reply:
Yes having only a single location for documentation would be my preference as well. If we really wanted to get fancy I wouldn't mind if it originally lived in the board directory and the docs system pulled it in.

Reply:
@dagar @davids5 The current docs system can pull in *whole documents* (only) from either local file system or github as documented [here](https://toolchain.gitbook.com/templating/conrefs.html).

The caveats are:
- Pulling from github is SLOW. Expect minutes added to build times.
- Source docs will need to be exposed for translation.

If this were needed it might be best to fetch the files ourselves into a local path as part prior to building.

So if this is something that you're still interested in doing, let me know what you want to pull in and we can look at the best approach given cost vs benefits.